src/components/HdExpandText.vue

```diff
@@ -121,6 +121,23 @@ export default {
     toggleExpanded() {
       this.isExpanded = !this.isExpanded;
     },
+    getLinesCount() {
+      if (typeof this.lines === 'number') {
+        return this.lines;
+      }
+
+      let linesCount = this.linesFallback;
+
+      Object.keys(this.lines).forEach((breakpoint) => {
+        if (!mediaMatches(breakpoint)) {
```

Comment:
For future improvement, it might make sense, to be able to pass breakpoints as prop here as well and then fallback to the main ones. Just to keep behavior consistent. If you pass `lines` as object 

```javascript
{
xl: 4,.
l: 3,
}
```

but don't set global endpoints, what would happen?

Reply:
Agreed, I will make a not to add this enhancement. Right now it would simply fall back to the `linesFallback`, but it would be nice to be able to set specific breakpoints since the component relies on them.